portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,22 +20,29 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.util.ArrayList;
+import java.util.HashMap;
+// import java.util.ArrayList;
```

Comment:
nit - can you remove all the commented out code? let's not check in code that unused.

Reply:
Ok will do